src/main/webapp/css/main.css

```diff
@@ -15,8 +15,12 @@
  */
 
 /* Prevent the page from becoming too wide. */
```

Comment:
Seems like this comment is not in the wrong place. Please place it right above the piece of code that the comment is referring too.

Reply:
Ok, I fixed the error and created a new pull request.